### ISSUE TITLE
Integrate with cweagans/composer-patches

### ISSUE THREAD

**Issue by jameswilson** [pantheon-systems/terminus-clu-plugin#11]
Many Drupal 8 sites use cweagans/composer-patches to apply patch files to contrib modules.

The problem is that the PR that gets created hides the fact that a patch didnt apply (buried in the extended notes of the commit message).

In my opinion, the PR should have some sort of comment that makes it very clear that the PR needs manual intervention when a patch doesn't apply.

Could be as simple as parsing the output of `composer update` and looking for the text string: `"Could not apply patch! Skipping. The error was: Cannot apply patch [some url]"`

**Comment by aaronbauman** [comment]
Have you run into this issue since https://github.com/pantheon-systems/terminus-clu-plugin/issues/8 was merged?

I agree, there should be some explicit flag(s) here on how to proceed when patching fails

**Comment by jameswilson** [comment]
This happend to us last week sometime, but #8  was merged 2 months ago.  

It's not clear to me exactly how to set the version of the `terminus-clu-plugin` to know whether the circleci job is not running the latest version.

I searched my entire codebase, and it turned up nothing:
```
Searching 34821 files for "terminus-clu-plugin"
0 matches
```

**Comment by jameswilson** [comment]
Hi @aaronbauman,  apologies for the delayed response. I'll admit I had to investigate this more closely and even now I dont clearly understand how this package is being included.

Our pantheon project is based off of https://github.com/pantheon-systems/example-drops-8-composer  and we have a verbatim copy of the `.circleci/config.yml` from there.

That file loads a docker image from: `quay.io/pantheon-public/build-tools-ci:6.x`  and then runs the composer_lock_updater job:

https://github.com/pantheon-systems/example-drops-8-composer/blob/83c0cb69fe098bf1d1eef7dd9d51833b70c9e97e/.circleci/config.yml#L234-L251

If I understand correctly this was fixed here: https://github.com/pantheon-systems/docker-build-tools-ci/pull/65  on the 6.x branch, and in theory should have been available to us already, so I don't know why we're still seeing this issue months later.
